osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -150,7 +150,7 @@ public void ResetOnlineInfo()
 
         public bool EpilepsyWarning { get; set; }
 
-        public bool SamplesMatchPlaybackRate { get; set; } = true;
+        public bool SamplesMatchPlaybackRate { get; set; }
```

Comment:
probably should trim

https://github.com/ppy/osu/blob/4a129ac0bf1fb149b3891a1f74a7f7b0d5dcf9ca/osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs#L195

in that case

Reply:
I'm still a bit weirded out as I believe the more sane default, at least in my eyes, is `true`. But let's give the mappers benefit of doubt because I don't have the energy to discuss a default setting like this further.